src/pages/Livestream/LiveRoomPage.css

```
@@ -1,1218 +1,1344 @@
+    align-items: center;
+    gap: 8px;
+    width: 100%;
+}
+
+/* Host mic button — idle state */
+.lr-host-mic-btn {
+    position: relative;
+    display: inline-flex;
+    align-items: center;
+    gap: 8px;
+    border: none;
```

Stylelint (17.7.0)

[error] Duplicate property "border" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)